laser_slam_ros/src/laser_slam_worker.cpp

```diff
@@ -208,6 +213,39 @@ void LaserSlamWorker::scanCallback(const sensor_msgs::PointCloud2& cloud_msg_in)
   }
 }
 
+bool LaserSlamWorker::getLaserTracksServiceCall(
+    laser_slam_ros::GetLaserTrackSrv::Request& request,
+    laser_slam_ros::GetLaserTrackSrv::Response& response) {
+  std::vector<std::shared_ptr<LaserTrack>> laser_tracks =
```

Comment:
The underlying IncrementalEstimator and LaserTrack are thread safe, though. Shoudn't that suffice because I only access those?